### PR TITLE
created a SL file and yaml file for add nic

### PR DESCRIPTION
**Pull request by manjoosh12**
Added SL and yaml files for the add nic operation

### CODE REVIEW

**Comment by lrevnic** [comment]
Maybe this is causing the CI/CD to fail:
[ERROR] 2020/06/12 10:09:27 LoggingServiceImpl -  
[ERROR] 2020/06/12 10:09:27 LoggingServiceImpl - ------------------------------------------------------------ 
[ERROR] 2020/06/12 10:09:27 LoggingServiceImpl - Exception: Test case with the name: **testDeleteNICSuccess** already exists. Test case name should be unique across the project

File: content/io/cloudslang/nutanix/prism/nics/add_nic.sl [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+########################################################################################################################
+#!!
+#! @description: Nic successfully Added to the VM. A VM NIC must be associated with a virtual network.
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/nics/add_nic.sl:17]: Change operation description.

File: content/io/cloudslang/nutanix/prism/nics/delete_nic.sl [hunk 2]
```diff
@@ -207,7 +207,7 @@ operation:
         private: true 
     
   java_action: 
-    gav: 'io.cloudslang.content:cs-nutanix-prism:1.0.0-RC6'
+    gav: 'io.cloudslang.content:cs-nutanix-prism:1.0.0-RC8'
```
Comment (patmante) [inline content/io/cloudslang/nutanix/prism/nics/delete_nic.sl:210]: update the GAV to RC9 for all operations.

File: content/io/cloudslang/nutanix/prism/nics/add_nic.sl (hunk 1 above)
Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/nics/add_nic.sl]: changed the description

File: content/io/cloudslang/nutanix/prism/nics/delete_nic.sl (hunk 2 above)
Comment (manjoosh12) [reply content/io/cloudslang/nutanix/prism/nics/delete_nic.sl]: upadted

**Pull request merged**